netket/sampler/metropolis.py

```diff
@@ -79,7 +81,7 @@ def n_steps(self) -> int:
     @property
     def n_accepted(self) -> int:
         """Total number of moves accepted across all processes since the last reset."""
-        res, _ = mpi.mpi_sum_jax(self.n_accepted_proc)
+        res, _ = mpi.mpi_sum_jax(jnp.sum(self.n_accepted_proc))
```

Comment:
the sum for the acceptance ratio is taken here, the same place were we already did it for mpi

Reply:
good. so can you finalise this pr?